thrift/binary_protocol.go

```diff
@@ -131,7 +131,6 @@ func (p *TBinaryProtocol) WriteMessageBegin(ctx context.Context, name string, ty
 		e = p.WriteI32(ctx, seqId)
 		return e
 	}
-	return nil
```

Comment:
Why these changes?

Reply:
could also be done in another PR, but that is unreachable code

Reply:
could also be done in a separate PR, but that is unreachable code
My linter was screaming

Reply:
Ok